alexa/request/request.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Alexa\Request;
+
+use RuntimeException;
+use InvalidArgumentException;
+use DateTime;
+
+use Alexa\Request\Certificate;
+use Alexa\Request\Application;
+
+class Request {
+
+	public $request_id;
+	public $timestamp;
+	/** @var Session */
+	public $session;
+	public $data;
+	public $raw_data;
+	public $applicationId;
+
+	/**
+	 * Set up Request with RequestId, timestamp (DateTime) and user (User obj.)
+	 * @param type $data
+	 */
+	public function __construct( $raw_data, $applicationId = null ) {
```

Comment:
Two different formats for var naming is weird. I've seen this elsewhere I think. Should be underscores generally in PHP, not camel case. Unless there's some alexa API related reason. Not a huge deal.